### PR TITLE
[13.0][MIG] hr_attendance_autoclose

### PR DESCRIPTION
**Pull request by jarroyomorales**
Migration to v13 of `hr_attendance_autoclose`

Superseedes https://github.com/OCA/hr/pull/779

@etobella @AaronHForgeFlow @Saran440 @kittiu

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Code + functional :+1:

**Review by kittiu — APPROVED** [review]

File: hr_attendance_autoclose/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "HR Attendance Auto Close",
+    "version": "13.0.1.1.0",
```
Comment (Saran440) [inline hr_attendance_autoclose/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

**Review by Saran440 — COMMENTED** [review]
Functional Test.
If I add `Attendance Maximum Hours Per Day` less than 1 hour. Function is not working

Code Review.

**Comment by jarroyomorales** [comment]
@Saran440 Module version updated

I've tested with maximum time < 1 hour and it is working as expected, can you provide an specific example? Thank you!

**Comment by Saran440** [comment]
@jarroyomorales My step to error
1. Used code with https://github.com/tegin/hr-attendance.git `branch` 13.0-mig-hr_attendance_autoclose
2. Create new database with demo data and Install module hr_attendance_autoclose
3. Go to Settings > Users & Companies > Companies > add value Attendance Maximum Hours Per Day = 0.5 (30 minute)
Selection_004
4. Check In and change time on local + 1 hour
Peek 2020-07-22 19-25


but if I setting 1 hour in company, It will work (Sometimes not work??)

**Comment by jarroyomorales** [comment]
@Saran440 There is a cron closing the attendances, maybe you didnt give enough time to it to execute?

**Comment by Saran440** [comment]
@jarroyomorales Oh, Okay I see. This module used scheduled action. Thank you for your advise.
Function test :+1:

**Review by Saran440 — APPROVED** [review]

**Review by etobella — APPROVED** [review]

**Pull request merged**